src/Native/configure.cmake

```diff
@@ -317,6 +317,14 @@ check_symbol_exists(
     "sys/syslimits.h"
     HAVE_OPEN_MAX)
 
+if (CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
On OSX, it is recommended to use the GSS framework. But probably @bartonjs comment about not checking for Linux still holds